freesound/settings.py

```diff
@@ -259,6 +259,13 @@
 # local settings if needed ;)
 DATA_URL = "/data/"
 
+# Locations where sounds and previews will be mirrored (if specified)
+# If locations do not exist, they will be created
+MIRROR_DISK_LOCATIONS = {
+    'SOUNDS': None,  # mirrors contents of SOUNDS_PATH
+    'PREVIEWS': None,  # mirrors contents of PREVIEWS_PATH
+}
+
```

Comment:
It'd be a good idea to have a setting here to turn it on and off, and check the setting in the copy util